### PR TITLE
Muting users

### PR DESCRIPTION
**Pull request by Simber1**
Mutes users by deleting their messages

### CODE REVIEW

**Review by Danial-Q — DISMISSED** [review]

**Comment by Simber1** [comment]
Might add a way to add users to mutedUsers before merging

**Comment by Simber1** [comment]
Closes #9

File: commands/mute.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+const {modRole} = require('../config.json');
+
+module.exports = {
+	name: 'mute',
+	description: 'Muting users',
+	guildOnly: true,
+	execute(message, args) {
+        var fs = require('fs');
```
Comment (Danial-Q) [inline commands/mute.js]: const {readFileSync, writeFile} = require ('fs');

File: commands/mute.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            const mutedJSONString = fs.readFileSync('muted.json','utf8');
+            console.log("String: "+mutedJSONString);
+
+            var mutedJSON = JSON.parse(mutedJSONString);
+            console.log("Keys = ",Object.keys(mutedJSON[0]));
+
+
+            args[0] = args[0].replace('<','');
+            args[0] = args[0].replace('>','');
+            args[0] = args[0].replace('@','');
```
Comment (Danial-Q) [inline commands/mute.js]: There must be a cleaner way of doing this, if not we should make a function to add some characters to a string before and after it, would be useful for things like mentioning users and such

File: commands/mute.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+                        message.reply("User already muted");
+                        return;
+                    }
+                    i++
+                }
+
+                const reason = args.splice(1).join(" ");
+                console.log(reason);
+
+                var d = new Date();
```
Comment (Danial-Q) [inline commands/mute.js]: Moment will be cleaner and a bit more robust

File: commands/unmute.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
+const {modRole} = require('../config.json');
+
+module.exports = {
+	name: 'unmute',
+	description: 'Muting users',
+	guildOnly: true,
+	execute(message, args) {
+        var fs = require('fs');
```
Comment (Danial-Q) [inline commands/unmute.js]: Just import the bits you need rather than the entire module, same as above

File: commands/mute.js (hunk 2 above)
Comment (Danial-Q) [reply commands/mute.js]: Try `message.mentions.user` or `message.mentions.members.first()`

File: commands/mute.js (hunk 3 above)
Comment (Danial-Q) [reply commands/mute.js]: Or just use `message.createdAt`

**Pull request closed without merging**